.github/workflows/tests.yml

```diff
@@ -274,7 +266,7 @@ jobs:
           environment-file: ci/environment-dashboard.yml
 
       - name: Generate dashboards
-        run: python dashboard.py -d benchmark.db -o static -b coiled-latest-py3.9 coiled-0.1.0-py3.9
+        run: python dashboard.py -d benchmark.db -o static -b coiled-latest-py3.9 coiled-0.2.1-py3.9
```

Comment:
I think this change is needed, but I can be wrong, 
@crusaderky please correct me if I'm wrong.

Reply:
Yes. The intent is to show the differences of `latest` and `upstream` vs. the latest published version.